src/main/resources/db/changelog/insert_default_mailing_template.sql

```diff
@@ -27,7 +27,11 @@ $endif$
 $if(blogsWithHomePage)$
 Nowo dodane blogi: <br/>
 $blogsWithHomePage.keys: { key |
-- <a href="$blogsWithHomePage.(key)$">$key.author$</a><br/>
+$if(blogsWithHomePage.(key).available)$
```

Comment:
No no no no :) You must create new changeset with new sql file updating *only* default mailing template.

Changesets in Liquibase are immutable by design.